Softeq.XToolkit.Bindings/BindingGeneric.cs

```diff
@@ -441,16 +441,14 @@ public override void ForceUpdateValueFromTargetToSource()
 
             if (string.IsNullOrEmpty(eventName))
             {
-                throw new ArgumentNullException("eventName");
+                throw new ArgumentNullException(nameof(eventName));
             }
 
             var type = PropertySource.Target.GetType();
             var ev = type.GetRuntimeEvent(eventName);
             if (ev == null)
```

Comment:
possible, but after check, I think we need to keep that as is because it is more comfortable to detect null argument during debugging (most frequent case)